public/stylesheets/style.css

```diff
@@ -1,11 +1,8 @@
-body {
-    font-family: 'Arial' !important
-}
-
-
 /*hero image*/
+
 .jumbotron {
-    background: linear-gradient( rgba(0, 0, 0, 0.45), rgba(0, 0, 0, 0.45)), url(/images/hero_image.jpg) no-repeat center center fixed;
+    background-image: linear-gradient( rgba(0, 0, 0, 0.45), rgba(0, 0, 0, 0.45)), url(/images/hero_image.jpg);
```

Comment:
yes somehow when i use background-image instead of background it looks better on meta viewport. do you know why?